src/app/components/LanguageNavigation/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import CollapsibleNavigation from '#app/components/CollapsibleNavigation';
+import { TypographyScript } from '#app/models/types/theming';
+import Navigation from '#psammead/psammead-navigation/src';
+import { Direction, Services } from '#app/models/types/global';
+import sections from './constants';
+
+type Props = {
+  script: TypographyScript;
+  service: Services;
+  dir: Direction;
+};
+const LanguageNavigation = ({ script, service, dir }: Props) => (
+  <Navigation script={script} service={service} dir={dir}>
```

Comment:
We don't need any of these values as props - we can derive `service`, `script` and `dir` from the `ServiceContext`